static/js/upload.js

```diff
@@ -112,6 +116,9 @@ exports.upload_files = function (uppy, config, files) {
         );
         return;
     }
+    if (exports.get_item("markdown_preview_hide_button", config).is(":visible")) {
+        exports.get_item("markdown_preview_hide_button", config).trigger("click");
```

Comment:
Would it be better to extract the click handler as a function and call that function?  It seems more readable to call "compose.exit_preview_mode()" than trigger a click.

Reply:
The click handler for `#undo_markdown_preview` button in compose box (while composing a new message) is already extracted as a function [here](https://github.com/zulip/zulip/blob/ba366244421bd78a5465374e4b545a93a9baf763/static/js/compose.js#L1218) while the code for handling click on `#undo_markdown_preview` button while editing message is written directly inside the click handler [here](https://github.com/zulip/zulip/blob/ba366244421bd78a5465374e4b545a93a9baf763/static/js/click_handlers.js#L376) and extracting it as a function would create inconsistency among other similar code in `click_handler.js` file.

And even if we do extract this click handler as a function, we would require to write extra logic in `upload.js` to check whether the `config.mode` is `compose` or `edit` and call the corresponding click handler function accordingly.